### ISSUE TITLE
Excluding fields when writing JSON documents from Storm to Elasticsearch doesn't work

### ISSUE THREAD

**Issue by jinuland** [elastic/elasticsearch-hadoop#711]
Hi, I'm working with elasticsearch storm and using "es.mapping.exclude" configuration for excluding some of fields. 
Below are my configuration

```
Map localConf = new HashMap();
        localConf.put("es.input.json", "true");
        localConf.put("es.mapping.exclude", "@version,@timestamp,id");
        builder.setBolt(ES_BOLT_ID, new EsBolt("log-2016.03.03/logs", localConf), ES_BOLT_PARALLEL_HINT).addConfiguration(Config.TOPOLOGY_TICK_TUPLE_FREQ_SECS, 2).shuffleGrouping(SPOUT_ID);
```

I've tested that code by submitting to my storm cluster, but is doesn't actually exclude the fields, which I wanna remove from final document. 

I've used elasticsearch-hadoop 2.2.0-beta1 artifact

**Comment by costin** [comment]
This has been fixed through #649. Upgrade to the latest stable version (2.2.0) and try it out - beta1 is just that, a beta  (and an old one).

**Comment by jinuland** [comment]
thanks costin it helps me a lot!
